datetime/src/test/java/com/fasterxml/jackson/datatype/jsr310/deser/OffsetDateTimeDeserTest.java

```diff
@@ -811,6 +812,40 @@ public void testOffsetDateTimeMinOrMax() throws Exception
         _testOffsetDateTimeMinOrMax(OffsetDateTime.MAX);
     }
 
+    @Test
+    public void OffsetDateTime_with_offset_can_be_deserialized() throws Exception {
```

Comment:
Seems like we can merge this and one for zonedDateTime below into a separate test class like   `Xxx336Test.java` for their purposes and similar style

Reply:
Sure, I can consolidate these into a separate test class

Reply:
In this case, I think keeping them along with existing test makes sense: this is not new functionality but optimizing (and adding test coverage). So let's not create per-issue test classes.